jenkins/automation/release-process.groovy

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env groovy
+pipeline {
```

Comment:
Can you please provide a Jenkins job link for this .groovy

Reply:
Added tested job link. once approved, will create a job in release process.